### DISCUSSION TITLE
Port Library to .net framework 4.8

### DISCUSSION THREAD

**Discussion by r0man75** [Poushec/UpdateCatalogLib#4] [Ideas]
Hello!
I used the library with success within .NET 7. Would it be possible to back port this library so it can be used with .NET Framework 4.8?

Thanks a lot and best regards
Roman

**Comment by Poushec** [comment]
Hello! I'll try to port it and let you know.

**Comment by Poushec** [comment]
Hello again. 

Since this library does not have any .NET-Only dependencies, I figured there isn't much sense in keeping it .NET-Only :) 
I've changed the target framework to netstandard2.0 and added a bunch of tests to verify it does work on .NET Framework 4.8.

Here are the links to [Nuget](https://www.nuget.org/packages/Poushec.UpdateCatalogParser/3.0.0) and [Release](https://github.com/Poushec/Poushec.UpdateCatalogParser/releases/tag/v3.0.0).
